### ISSUE TITLE
`prohibited-package` gives false warnings in `org.eolang.string-test`

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#71]
In a unit test for `org.eolang.string`:

```
+package org.eolang
+tests

[] > foo
  true > @
```

I'm getting this warning:

```
org/eolang/string-tests.eo Package org.eolang is reserved for internal object only (prohibited-package)
```

I believe, we must avoid this message if `+tests` is present.

**Comment by h1alexbel** [comment]
@yegor256 assign me please

**Comment by yegor256** [comment]
@h1alexbel go ahead
